package.json

```diff
@@ -301,34 +301,34 @@
     "eslint-plugin-jsx-a11y": "6.10.0",
     "eslint-plugin-prettier": "5.2.1",
     "eslint-plugin-promise": "7.1.0",
-    "eslint-plugin-react": "7.35.2",
+    "eslint-plugin-react": "7.36.1",
     "eslint-plugin-security": "3.0.1",
     "eslint-plugin-sort-destructure-keys": "2.0.0",
     "eslint-plugin-sort-imports-es6-autofix": "0.6.0",
     "eslint-plugin-sort-keys-fix": "1.1.2",
     "eslint-plugin-sort-requires-fix": "2.1.2",
     "eslint-plugin-spellcheck": "0.0.20",
-    "husky": "9.1.5",
+    "husky": "9.1.6",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "jest-environment-node": "29.7.0",
-    "jsdom": "25.0.0",
+    "jsdom": "25.0.1",
     "lint-staged": "15.2.10",
     "lint-staged-shellcheck": "0.1.2",
     "livereload": "0.9.3",
-    "nodemon": "3.1.4",
+    "nodemon": "3.1.7",
     "pdf2pic": "3.1.3",
     "pixelmatch": "6.0.0",
     "pngjs": "7.0.0",
-    "postcss": "8.4.45",
-    "postcss-preset-env": "10.0.3",
+    "postcss": "8.4.47",
+    "postcss-preset-env": "10.0.5",
     "prettier": "3.3.3",
     "puppeteer": "23.0.0",
     "puppeteer-core": "23.0.0",
```

Comment:
Not directly related to what you've done here, but we have a comment in `dependency-updates.md`:

"As of 8/15/2024, we cannot update puppeteer or puppeteer-core beyond 22.13.1 because the latest release of @sparticuz/chromium only supports version 126 of chromium."

I think we can probably remove that. No big deal though. I can do that later this week as part of my dependency updates if that is easier.

Reply:
Please tackle it with your updates. Thanks!